manager/allocator/network.go

```diff
@@ -1071,6 +1072,16 @@ func (a *Allocator) procTasksNetwork(ctx context.Context, onRetry bool) {
 	}
 }
 
+// IsBuiltInNetworkDriver returns whether the passed driver is an internal network driver
+func (a *Allocator) IsBuiltInNetworkDriver(name string) bool {
+	return a.netCtx.nwkAllocator.IsBuiltInDriver(name)
+}
+
+// PredefinedNetworks returns the list of predefined network structures for a given network model
+func PredefinedNetworks() []networkallocator.PredefinedNetworkData {
```

Comment:
I haven't defined the require type yet, although I have code to do it in my POC, shall I pull that over?